CHANGELOG.md

```diff
@@ -66,6 +66,10 @@
 - Added support for `Series.dt.is_leap_year`.
 - Added support for `DataFrame.items`.
 - Added support for `Series.dt.floor` and `Series.dt.ceil`.
+- Added support for `DatetimeIndex` properties: `year`, `month`, `day`, `hour`, `minute`, `second`, `microsecond`,
```

Comment:
Can you also update the documentation (`indexing.rst` and `index_supported.rst`)?

Reply:
Sure, I have a separate JIRA SNOW-1570717 to take care of documentation. I think plan is to add separate files for datetime_index.

Reply:
I think it's best when the changelog is synchronized with the docs - but if it's a quick follow-on then I think it should be fine.

Reply:
I Agree. I believe this is a new class so we have added a separate JIRA to rake care of docs at multiple places. I have pulled this into next sprint to take care of all doc related files.